src/notes/patch v3.6.md

```diff
@@ -12,74 +12,74 @@
-- Fixed bug where tranq darts were sometimes dealing lethal damage
+- Fixed a bug where tranq darts were sometimes dealing lethal damage
 - Some bingo-relevant books are now color coded for easier recognition (The Man Who Was Thursday now has a red cover, and Jacob's Shadow is now purple).  This can be controlled by the Goal Textures setting.
 - NG+/WW reduced skills churn
   - NG+ does 2 skill downgrades instead of 3
   - WaltonWare crates have 33% fewer skill points
   - NG+ takes away 20% of your unspent skill points instead of 25%
 - Max Rando/NG+ prevent min_weapon_dmg from going too low, old range was 25% to 100%, now it's 40% to 90%
-- MaxRandoValPair keep more separation between min and max (like for the min/max weapon damage, or min/max skill costs)
-- Pool is much improved
-- Removed Vandenberg start location at front gate
+- `MaxRandoValPair` now has more separation between min and max (like for the min/max weapon damage, or min/max skill costs)
+- Pool tables are much improved
+- Removed Vandenberg start location at front gate (where you find Jock after you completed your goals in the non-Randomized game)
 - Added Vandenberg start location on roof of the comms building
 - Bingo boards generated for Mr. Page's Mean Bingo Machine now only have 1 free space instead of 5.
 - Randomly selected starting locations for New Game+ will try to avoid repeating recently used start locations more.  This also includes repeat UNATCO HQ or Hell's Kitchen starts (even across different visits)
 - WaltonWare crates will now give medkits depending on your start location and your maximum health
 - Added scaling and blackout options for scopes and binoculars, with a cool automatic "Fit to Screen" option that doesn't rely on fixed size textures and adapts to any screen resolution/ratio
-- Proper handling of Ford Schick unable to give you the aug upgrade can
+- Proper handling of Ford Schick being unable to give you the aug upgrade can when your inventory is full
 - Make sure Shannon is always present in M01, regardless of gender #1280
-- Change many "Kill Person X" goals to "Take Down Person X", allowing both kills and knockouts instead of just kills.  Bingo boards from older versions will be upgraded to this.
+- Many "Kill Person X" goals are changed to "Take Down Person X", allowing both kills and knockouts instead of just kills.  Bingo boards from older versions will be upgraded to this.
 
 # Bingo / WaltonWare / NG+ Minor Changes
 
 - NG+ crates subtract unspent skill points and credits instead of silently taking them away in NG+
   - (instead of NG+ silently taking away unspent skill points from the player, now it tries to remove them from the WW crate first)
 - Fewer datacubes in Area51 Entrance and Page (#1244 and #1245)
-- NG+ and DXRReduceItems respect 0% for WW Hardcore medkits/medbots
-- Hong Kong main quest bingo goals not given for early starts
+- NG+ and `DXRReduceItems` respect 0% for WW Hardcore medkits/medbots
```

Comment:
I'm actually not sure this one needs mentioning at all. It's basically a bug fix for a game mode that didn't exist before this release.

```suggestion
```